package.json

```diff
@@ -39,7 +39,7 @@
     "compile-json-stringify": "^0.1.2",
     "inquirer": "^8.2.4",
     "is-my-json-valid": "^2.20.0",
-    "moment": "^2.24.0",
+    "moment": "^2.29.3",
```

Comment:
Should this be removed?

Reply:
I think so

Reply:
We still use it for tests. I moved it to devDep, but I guess we can replace it with `luxon` there. https://github.com/fastify/fast-json-stringify/issues/428